packages/typescript-config/tsconfig.json

```diff
@@ -1,14 +1,14 @@
 {
-  "compileOnSave": false,
+  "$schema": "https://json.schemastore.org/tsconfig",
   "compilerOptions": {
     "esModuleInterop": true,
     "forceConsistentCasingInFileNames": true,
     "importHelpers": true,
     "incremental": true,
     "jsx": "react",
```

Comment:
❓ Should `react` be capitalized to match the rest of the configuration?

Reply:
@FarrisPandell

See my rant about Microsoft being _"incredibly inconsistent about string casing"_. As per my notes in PR description, I am using the schema's casing for all strings. And - for some reason - the `jsx` property currently uses `camelCasing`. Sigh...

https://json.schemastore.org/tsconfig

<img width="299" alt="image" src="https://github.com/user-attachments/assets/f2bc9155-dc31-4a99-ba9f-ca6f3f16fbfc">

From the same schema:

<img width="898" alt="image" src="https://github.com/user-attachments/assets/9b606409-d73d-415f-a316-ac2320a3823d">